src/backend/metal/src/command.rs

```diff
@@ -3000,6 +3004,33 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
             if let Some(command) = self.state.set_vertex_buffers() {
                 pre.issue(command);
             }
+            // Note: all VS resources past the `old_attribute_buffer_index` have to be re-bound
```

Comment:
What prevents this from happening with CS buffers too?

Reply:
the "attribute_index" is for vertex buffers only, so it's only relevant for VS stage